ckanext/dcatapchharvest/tests/fixtures/deprecated/dataset-publisher.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+         xmlns:dcat="http://www.w3.org/ns/dcat#"
+         xmlns:dct="http://purl.org/dc/terms/"
+         xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
+
+  <dcat:Dataset rdf:about="https://swisstopo/123">
+    <dct:publisher>
+      <rdf:Description>
+        <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#Description"/>
+        <rdfs:label>Bundesamt für Landestopografie swisstopo</rdfs:label>
+      </rdf:Description>
```

Comment:
you check here for the old format of publishers and use `lable` to create a new format, right?
as is given in ` publisher_deprecated = self._object_value(agent, RDFS.label)`

If so, why do we already have ` <dct:publisher>` here and not  `<dct:publishers>`?